config/locales/en.yml

```diff
@@ -66,10 +66,12 @@ en:
     numerator: "Patients with no visit in the last 3 months."
   diabetes_missed_visits_copy:
     reports_card_subtitle: "Diabetes patients in %{region_name} with no visit in the last 3 months"
-    numerator: "Diabetes patients with no visit in the last 3 months."
+    numerator: "Patients with no visit in the last 3 months."
```

Comment:
Intentional?

Reply:
yes. In the Figma design for DM reports and the htn cards also use this pattern